package.json

```diff
@@ -55,11 +54,13 @@
     "@react-native-picker/picker": ">= 2.2.1",
     "@react-navigation/bottom-tabs": "6.5.7",
     "@react-navigation/core": "6.4.8",
+    "@react-navigation/native": "6.1.6",
     "@react-navigation/stack": "6.3.16",
     "react": ">= 16.13.1",
     "react-native": ">= 0.63.3",
     "react-native-gesture-handler": "2.10.1",
     "react-native-keyboard-aware-scrollview": ">= 2.1.0",
+    "react-native-safe-area-context": "4.2.4",
```

Comment:
Does this need to be added since it wasn't part of `dependencies`? if so, should it also be added to `devDependencies`?

Reply:
I don't use it directly in Hyperview so I didn't have it as a dependency or dev dependency. The navigators need it. It's in the demo package so I figured it was better to explicitly mark it as a peer dependency.